codeship-services.yml

```diff
@@ -40,8 +40,8 @@ e2e:
     - /dev/shm:/dev/shm
 
 db:
-  image: mdillon/postgis:9.6-alpine
-  cached: true
+  image: mdillon/postgis:10-alpine
+  cached: false
```

Comment:
Will switch this back to cached as soon as I'm sure it's the correct distro to use 👍